### ISSUE TITLE
Unable to connect from Safari browser

### ISSUE THREAD

**Issue by wornish** [rkaczorek/astroberry-server#20]
Loaded the latest version of astroberry today on to my RPi3B+

Trying to access from my iMac running OSX 10.14.3

ssh connection works fine. via Wifi
VNC connection also works fine but is a little laggy.

When I try and connect via my safari browser I get asked to connect and when I enter the password I get the error message below.

Also tried with Opera browser and get the same message.

![screenshot 2019-03-07 at 14 04 28](https://user-images.githubusercontent.com/964548/53963900-188c1480-40e6-11e9-8ba6-84f4e8aa10a1.jpg)

**Comment by rkaczorek** [comment]
javascripts are not loading properly - just refresh the page until it disappears, you should be fine

**Comment by wornish** [comment]
Thanks for getting back. 

I tried it from my Windows 10 laptop and it works fine, which is where I will be using it anyway, so that’s good.


I have now  tried multiple times from Safari on my iMac and I still get the same error.
Seems like its something strange going on with that.  The connect window appears and it lets me enter my password then when I click connect I get the error message.


  I have played with EKOS and it looks promising.


Regards
Dave
